dxp/wysiwyg.md

```diff
@@ -0,0 +1,33 @@
+# WYSIWYG Editors
+
+We have recently worked on reducing variability in the WYSIWYG writing experience. After analyzing current usage of the editors the decision was made to provide a more cohesive and comprehensive experience for each specific use case. Documenting the **WYSIWYG** editors that are available in [Liferay DXP](https://github.com/liferay/liferay-portal) can be useful.
+
+At the time of writing this document, there are three different **WYSIWYG** editors in [Liferay DXP](https://github.com/liferay/liferay-portal):
+
+- [AlloyEditor](https://alloyeditor.com)
+- [CKEditor 4](https://ckeditor.com/ckeditor-4/)
+- [TinyMCE](https://www.tiny.cloud/) - disabled by default
+
+Editors are usually instantiated in jsp editor taglib and in js files. 
```

Comment:
How? Where? Show examples.

Reply:
Sure, I will add examples on "how and where" in the next document iteration.